_data/content/projects/fr.yml

```diff
@@ -5,228 +5,137 @@
 page_data:
```

Comment:
After rebasing with main check the unused code, as example, lines 8 to 12 are removed, check it from the spanish or english version